cli/options.json

```diff
@@ -190,7 +190,7 @@
     "description": [
       "Overrides the stack size. Only relevant for incremental GC",
       "or when using a custom runtime that requires stack space.",
-      "Defaults to 0 without and to 16384 with incremental GC."
+      "Defaults to 0 without and to 65536 with incremental GC."
```

Comment:
This can be a problem for wasm4 and wasm3 runtimes which has only one page 64kb for everything

Reply:
Can try to use a smaller number, it's just that 16384 isn't large enough for the compiler. I guess restricted environments would want to configure a smaller value anyway when using the full runtime, hmm.

Reply:
32768 seems to be sufficient. The alternative is we leave it at 16384 and configure bootstrapping differently.